ci/run_clang_tidy.sh

```diff
@@ -1,9 +1,16 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 set -e
 
+ENVOY_SRCDIR=${ENVOY_SRCDIR:-$(cd $(dirname $0)/.. && pwd)}
```

Comment:
where is this used?

Reply:
https://github.com/envoyproxy/envoy/blob/master/ci/run_clang_tidy.sh#L26